modules/navier_stokes/test/tests/ins/IdealAMG/2d_diffusion_amg.i

```diff
@@ -0,0 +1,48 @@
+[Mesh]
```

Comment:
I'm guessing you didn't mean for this file to get in here?

Reply:
Beautiful cask branch?

Reply:
It is disaster!!